docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,533 @@
-there has been no data loss, it just means that tool was able to complete its job.
+  Before you run the ``repurpose`` command, you need to stop CrateDB.
 
-Detach a node from a cluster
-----------------------------
+To find and remove excess shard data and table metadata, run the following
+command and answer ``y`` at the confirmation step:
 
-When a node joins a cluster initially, the unique cluster id will be stored in
-its nodes metadata. This prevents the node from joining a cluster with a different
-cluster id. There are situations where it necessary to reset the cluster id from
-a node. When a cluster is not recoverable, it might be worth moving nodes
-to a new cluster. This is in particular useful when the voting configuration
-of a node is reset, and a new cluster is bootstrapped.
+.. code-block:: console
 
-It is unsafe for nodes to move between clusters, because different clusters have
-completely different cluster metadata. There is no way to safely merge the
-metadata from two clusters together.
+    sh$ node$./bin/crate-node repurpose
 
-To protect against inadvertently joining the wrong cluster, each cluster creates
-a unique identifier, known as the cluster UUID, when it first starts up. Every
-node records the UUID of its cluster and refuses to join a cluster with a
-different UUID.
+    Found 2 tables (2 shards and 2 table metadata) to clean up.
+    Use -v to see a list of paths and tables affected.
+    Node is being repurposed as no-master and no-data. Clean-up of table data
+    is performed.
 
-However, if a node’s cluster has permanently failed then it may be desirable
-to try and move it into a new cluster. The ``crate-node detach-cluster`` command
-lets you detach a node from its cluster by resetting its cluster UUID.
-It can then join another cluster with a different UUID.
+    Do you want to proceed?
 
-For example, after unsafe cluster bootstrapping you will need to detach all
-the other surviving nodes from their old cluster so they can join the new,
-unsafely-bootstrapped cluster.
+    Confirm [y/N] y
+    Node successfully repurposed to no-master and no-data.
 
+.. _unsafe-bootstrapping-cluster:
 
-.. WARNING::
+Unsafely bootstrapping a cluster
+================================
 
-  Execution of this command can lead to arbitrary data loss. Only run this tool
-  if you understand and accept the possible consequences and have exhausted all
-  other possibilities for recovery of your cluster.
+Using the ``unsafe-bootstrap`` command, you can unsafely override the voting
+configuration of a cluster as if performing another cluster-bootstrapping
+process. This operation is useful if there is at least one remaining
+master-eligible node, but it is not possible to restart a majority of the nodes.
 
-The sequence of operations for using this tool are as follows:
+If you have a choice of nodes on which to run the ``unsafe-bootstrap`` command,
+choose the one with the highest term. The node's terms increase with
```

Comment:
can you define "term"? link to to docs, etc

how do you find this value? etc

Reply:
Can we add the term TERM to the appendix and then point to the description there? https://crate.io/docs/sql-99/en/latest/appendices/f.html#v